#controller_joint_names: ['', 'leg03', 'high03', 'low03', 'leg02', 'high02', 'low02', 'leg01', 'high01', 'low01', ]

joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

# Position Controllers ---------------------------------------
leg03_position_controller:
    type: effort_controllers/JointPositionController
    joint: leg03
    pid: {p: 100.0, i: 0.005, d: 0.1}
    
high03_position_controller:
    type: effort_controllers/JointPositionController
    joint: high03
    pid: {p: 40.0, i: 0.005, d: 0.1}
     
low03_position_controller:
    type: effort_controllers/JointPositionController
    joint: low03
    pid: {p: 30.0, i: 0.005, d: 0.1}

leg02_position_controller:
    type: effort_controllers/JointPositionController
    joint: leg02
    pid: {p: 100.0, i: 0.005, d: 0.1}
    
high02_position_controller:
    type: effort_controllers/JointPositionController
    joint: high02
    pid: {p: 40.0, i: 0.005, d: 0.1}
    
low02_position_controller:
    type: effort_controllers/JointPositionController
    joint: low02
    pid: {p: 30.0, i: 0.005, d: 0.1}

leg01_position_controller:
    type: effort_controllers/JointPositionController
    joint: leg01
    pid: {p: 100.0, i: 0.005, d: 0.1}
    
high01_position_controller:
    type: effort_controllers/JointPositionController
    joint: high01
    pid: {p: 40.0, i: 0.005, d: 0.1}
    
low01_position_controller:
    type: effort_controllers/JointPositionController
    joint: low01
    pid: {p: 30.0, i: 0.005, d: 0.1}
    
